examples_v20x/exti_blink/exti_blink.c

```diff
@@ -0,0 +1,60 @@
+#include "ch32fun.h"
+
+// specify interrupt line
+void EXTI1_IRQHandler( void ) __attribute__((interrupt));
+
+// set state of the LED on PA10
+int state = 1;
+
+int main()
+{
+	SystemInit();
+
+	funGpioInitAll();
```

Comment:
Please add a comment what clocks/enables you are leveraging from this call.

Reply:
For the EXTI handler I am not sure what I write. Is it just that it enables the line 1 external interrupt?

For the funGpioInitAll() I have written:
```
/*
The enables that end up being used in this code are
GPIOA  (port A enable)
AFIO   (alternate function enable)
AFIOEN (alternate function clock enable)
*/
```

Reply:
Looks good.